client/components/App.jsx

```diff
@@ -1,9 +1,34 @@
 import React from 'react'
+import { connect } from 'react-redux'
 
-// This might need to be turned into a stateful component (aka container)
-const App = () => (
-  <div className='app'>
-  </div>
-)
+import Header from './Header'
+import Listing from './Listing'
+import Cart from './Cart'
 
-export default App
+const App = (props) => {
+  const path = props.navigation
+  if (path === 'home'){
+    return (
+      <div>
+        <Header />
+        <Listing />
+      </div>
+    )
+  }
+  if (path === 'cart') {
+    return (
+      <div>
+        <Header />
+        <Cart />
+      </div>
+    )
```

Comment:
since you are only changing one component you could use a ternary operator instead

```
return (
  <div>
    <Header />
    { path == 'cart' ? <Cart> : <Listing> }
  </div>
  )

sort of thing